### DISCUSSION TITLE
A bit of clarification please...

### DISCUSSION THREAD

**Discussion by Alb673** [linux-surface/linux-surface#1334] [Q&A] [answered]
Hi everyone,
I just found this site after watching a video of Linux installed on a GO2 and  would like a bit of clarification. 

Under Wiki, there is a list of supported device, but I noticed that some are written in red.

Are those NOT supported or not fully supported? Ex Pro 5(5th gen 2017) or Pro 6 and others.

I have been looking on Ebay for a cheap Pro 4 (written in blue),  but Microsoft confirmed that screen flickering is not fixable with a screen replacement, so I now need to look for another model.

Apologies if this question has been already addressed, didn't find any result in the search.
Thanks in advance for any advice.
Alberto

**Comment by qzed** [comment, marked as answer]
I assume you mean the red links in the sidebar and/or the list of devices at the top of [this](https://github.com/linux-surface/linux-surface/wiki/Supported-Devices-and-Features) page. That just means that there is no dedicated wiki page for those. That doesn't really reflect the support status, it just says that no one has written down any specific quirks or tips for that device yet. As long as it's in that list as "supported", it's at least decently supported (or being worked on). You can check [these tables](https://github.com/linux-surface/linux-surface/wiki/Supported-Devices-and-Features#surface-tablets) to see the details .

**Reply by Alb673** [reply]
Hi qzed, yes that's exactly what I mean. Thanks for the clarification. 
I'm glad to have more options to choose from :)
Happy New Year
